pygmt/src/meca.py

```diff
@@ -170,30 +172,30 @@ def meca(
         - ``"principal_axis"`` (principal axis)
 
         Ignored if ``spec`` is a dictionary or pd.DataFrame.
-    component: str
+    component : str
         The component of the seismic moment tensor to plot.
 
         - ``"full"``: the full seismic moment tensor
         - ``"dc"``: the closest double couple defined from the moment tensor
           (zero trace and zero determinant)
         - ``"deviatoric"``: deviatoric part of the moment tensor (zero trace)
-    longitude: int, float, list, or 1-D numpy array
+    longitude : int, float, list, or 1-D numpy array
         Longitude(s) of event location. Must be the same length as the
         number of events. Will override the ``longitude`` values
         in ``spec`` if ``spec`` is a dict or pd.DataFrame.
-    latitude: int, float, list, or 1-D numpy array
+    latitude : int, float, list, or 1-D numpy array
         Latitude(s) of event location. Must be the same length as the
         number of events. Will override the ``latitude`` values
         in ``spec`` if ``spec`` is a dict or pd.DataFrame.
-    depth: int, float, list, or 1-D numpy array
+    depth : int, float, list, or 1-D numpy array
         Depth(s) of event location in kilometers. Must be the same length as
```

Comment:
```suggestion
        Depth(s) of event location(s) in kilometers. Must be the same length as
```

Reply:
Thanks! I updated some more occurrences in commit https://github.com/GenericMappingTools/pygmt/commit/e2b1a6df3a5adf3d8ca25a0fb29d12aa5367da75.

However, I am wondering why this commit does not show up in this PR. For me I looks like I committed to the right branch (please see https://github.com/GenericMappingTools/pygmt/compare/add-meca-aliases-E-G). Currently I am unfortunately not sure what I did wrong.